runtime/opt/taupage/init.d/06-update-sysctl.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import subprocess
+
+from taupage import configure_logging, get_config
+
+SYSCTL_WHITELIST = ['vm.dirty_background_ratio', 'vm.dirty_ratio', 'vm.overcommit_memory', 'vm_swappiness',
+                    'vm.overcommit_ratio']
+SENZA_SYSCTL_CONF = '/etc/sysctl.d/99-senza.conf'
+
+
+def write_file(path, content):
```

Comment:
The reason for me to put the whitelist here is to add in the visibility of the whitelist.
I don't have a strong opinion on this or inline though.